tf_encrypted/keras/backend.py

```diff
@@ -0,0 +1,59 @@
+"""TFE Keras backend"""
+import threading
+
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import array_ops
+from tensorflow.python.keras.backend import get_graph, name_scope
+
+import tf_encrypted as tfe
+
+# This is a thread local object that will hold the default internal TFE session
+# used by TFE Keras. It can be set manually via `set_session(sess)`.
+_SESSION = threading.local()
+
+
+def get_session(op_input_list=()):
+  """Returns the session object for the current thread."""
+  global _SESSION
+  default_session = ops.get_default_session()
+  if default_session is not None:
+    session = default_session
+    # If the default session is not a TFE Session, create one
+    if not isinstance(session, tfe.Session()):
+      session = tfe.Session()
```

Comment:
if the session is created here it won't be reused since it's not stored back onto `_SESSION`, don't think this is intended behaviour

Reply:
ah good catch!